plan/logical_plan_builder.go

```diff
@@ -1453,7 +1453,11 @@ func (b *planBuilder) buildUpdate(update *ast.UpdateStmt) LogicalPlan {
 		return nil
 	}
 	p = np
-	updt := Update{OrderedList: orderedList}.init(b.allocator, b.ctx)
+
+	updt := Update{
+		OrderedList: orderedList,
+		Ignore:      update.Ignore,
```

Comment:
can we change the variable name `Ignore` to a more specific one, like `IgnoreError` ?

Reply:
I think the variable should be same with the Insert.

Reply:
yes, we should change the naming in `Insert`, too

Reply:
OK，I think IgnoreErr would be more specific. I'll change the variable name.

Reply:
ok

Reply:
Done